source/renderer/zettlr-editor.js

```diff
@@ -778,6 +778,16 @@ class ZettlrEditor {
     return this
   }
 
+  /**
+   * Closes all open files
+   */
+  closeAll () {
+    this._currentHash = null
```

Comment:
I am pretty sure we need to remove this statement, as it should be reset whenever a tab is being closed (which is happening if you send a file-close command – what is happening is that (a) renderer sends command (b) main checks if we can close the file and perform a cleanup (c) main sends a command to actually close the file to the renderer (d) the editor will reset the `this._currentHash` to `null` if the last tab has been closed.

Reply:
I added that specifically to handle an edge condition. Steps to reproduce:
1. Open 3-4 tabs
1. Close Zettlr
1. Relaunch Zettlr
1. Zettlr will come up with previous tabs with one tab highlighted.
1. Right click on the highlighted tab and select close all

You'll see that the last tab still remains open. 

I added setting `_currentHash` to `null` to handle this. But your comment made me dig in a bit deeper. To me it looks like there is a bug elsewhere in the system. When closeAll fires off  IPC for `n` files, only `n-1` calls to close() are received causing last file to remain open. This is happening only when you close Zettlr with many tabs and relaunch it. I am not sure what might be causing this.

Reply:
> To me it looks like there is a bug elsewhere in the system.

Very likely!

> When closeAll fires off IPC for n files, only n-1 calls to close() are received causing last file to remain open. This is happening only when you close Zettlr with many tabs and relaunch it. I am not sure what might be causing this.

Thank you for digging into this! Let's suggest the following, as we still have time until the next iteration will be released:

1. You remove that statement, as it's not supposed to be in there
2. I will open a bug report with your findings so that we can handle this correctly :) 

Thank you very much for this detailed bug description, because it'll make removing the bug very easy!

Reply:
Thank you. I have removed that line.